doc/release/1.2.1-notes.rst

```diff
@@ -0,0 +1,16 @@
+==========================
+SciPy 1.2.1 Release Notes
+==========================
+
+.. contents::
+
+SciPy 1.2.1 is a bug-fix release with no new features compared to 1.2.0.
```

Comment:
I would add something like "Most importantly it does solve the issue that 1.2.0 cannot be installed from source on Python 2.7 because of non-ascii character issues".